### ISSUE TITLE
Cron resets early during quests

### ISSUE THREAD

**Issue by loveallama** [HabitRPG/habitica#3296]
During more than one quest, my cron time has been automatically reset to 12am instead of 1am or 2am. Sometimes those extra couple hours make a big difference.

My party also had a lot of trouble during the Vice Awakens quest, where a couple of us died but continued to take damage though we did not deal damage or receive rewards.

Neither of these are a top priority, just mentioning bugs. Thanks

**Comment by deilann** [comment]
Are you sure you did not deal damage or receive rewards? Currently, death
has _no_ impact on quests at all. It does not take you out of the running
in any way, shape, or form.

**Comment by Mavrimorn** [comment]
Following the directions to report a bug, as I have recently been having trouble with my cron.

This past Wednesday (July 16) I opened HabitRPG to find that I had not cronned from the day before. All my dailies were still checked from the day before, and the one daily that only opens on Wednesdays was still closed. Some time in the afternoon, they reset. I noticed the reset when I clicked on a specific daily of mine and was informed that I had died (I had 49 health before). The weekly assignment was still closed. The next day, Thursday, my cron was again late and now the Wednesday assignment was open. I did not die, but I lost health down to 17. Today, my cron was again late, and I was informed again that I died.

I have selected this thread because I recently (earlier this week, perhaps Tuesday, wish I could remember the exact day) joined a party for the first time and at the same time also joined a party quest.

Thank you for your time and effort.

**Comment by deilann** [comment]
Okay, so cron happens when you first sync with the server, not at your day
start. Are you clicking dailies and then you click a specific daily later,
and it happens? I'm kind of confused. Make sure to sync or refresh and see
if that triggers cron.

If you're in a party quest, what's more likely is that you're taking damage
later in the day from your party members. Go to your party page and
refresh; see if they've been dealing damage. :)

**Comment by Mavrimorn** [comment]
The daily I was clicking may have been the first that I would click in that particular session. Multiple refreshes/syncs earlier in the day would fail to produce a cron. I know it's not my party, because they are awesome and never accrued any damage. I was the only one who triggered damage during the boss battle.
The last time I was killed though (yesterday), I went to the settings page to reset my values. While there, I saw something I had not seen before called "Custom Day Start." For some reason it was set to 21:00 or 9 pm. I changed this to 2 (2 am) to see if that makes a difference. I wasn't able to tell today, due to not being near my computer all day long. I will check tomorrow.
If it does turn out to be the "Custom Day Start," is there a way to turn it off and just use the default for my time zone?
